common/src/main/scala/explore/schemas/odb/implicits.scala

```diff
@@ -240,6 +242,28 @@ object implicits {
       }
   }
 
+  implicit class PosAngleConstraintOps(val p: PosAngleConstraint) extends AnyVal {
+    def toInput: PosAngleConstraintInput =
```

Comment:
@toddburnside I guess this is why we could remove the encoder?

Reply:
That's my understanding at least.  This is doing what the encoder would have done.  Deleting the constraint when necessary is handled at a higher level?

Reply:
In explore it is an `Option[PosAngl]` when set to none (In the UI Unconstrained) it gets removed on the db via an `Input.orUnassign`

Reply:
> @toddburnside I guess this is why we could remove the encoder?

Yes, exactly. We can never just encode directly to the UI because the mutations require the *Inputs.